ide/coqide/coqide.ml

```diff
@@ -1598,12 +1601,16 @@ let build_ui () =
   menu debug_menu [
     item "Debug" ~label:"_Debug";
     item "Toggle breakpoint" ~label:"_Toggle breakpoint" ~accel:"F8"
-      ~callback:MiscMenu.toggle_breakpoint;
-    item "Continue" ~label:"_Continue" ~accel:"F9" ~callback:Nav.continue;
-    item "Step in" ~label:"Step in" ~accel:"F10" ~callback:Nav.step_in;
-    item "Step out" ~label:"Step out" ~accel:"<Shift>F10" ~callback:Nav.step_out;
+      ~stock:`MEDIA_RECORD ~callback:MiscMenu.toggle_breakpoint;
+    item "Continue" ~label:"_Continue" ~accel:"F9"
+      ~stock:`MEDIA_NEXT ~callback:Nav.continue;
+    item "Step in" ~label:"Step in" ~accel:"F10"
+      ~stock:`MEDIA_PLAY ~callback:Nav.step_in;
+    item "Step out" ~label:"Step out" ~accel:"<Shift>F10"
+      ~stock:`MEDIA_FORWARD ~callback:Nav.step_out;
```

Comment:
I don't see these icons on my system.  Are you sure they're available on all platforms?

![image](https://user-images.githubusercontent.com/1253341/230512308-4f3af8d3-052c-47b0-aea4-3641ff4e4a08.png)

Reply:
Here are the icons I have:
![coqide-icons-debug-help](https://user-images.githubusercontent.com/5684776/232521885-da79c551-bb5f-450f-8e44-da58e789b7c7.png)

> Are you sure they're available on all platforms?

I’m not a specialist of GTK, I don’t really like it (as I said above, wxWidgets follows my ideas and principles much better). The icons you have depend on your GTK icon theme; I personally use the GNOME icon theme (I never use GNOME though). I can’t provide much more support to people who don’t have the icons…! I’m using Manjaro Linux with Xfce and I can choose my icon theme in Xfce Settings Manager > Appearance > Icons. The first icon is stored on my system at: /usr/share/icons/gnome/24x24/actions/media-record.png

Reply:
I'm not sure the icons add much.  These don't convey the meaning of the operations well.  For example, your icon for "step in" is more appropriate for "continue".  Ideally we would make some of our own icons.  Or just omit them.

Reply:
I think it’s still better to have these icons when they work; it adds some colors, some flavor. Without them, the feature may look cold, distant, or even unmaintained.

> your icon for "step in" is more appropriate for "continue".

I don’t think so, because “Continue” may go immediately to the end of the buffer, while “Step in” will advance only of a minimal step.

> Ideally we would make some of our own icons.

Indeed, if it is a matter of a few shapes, I’m able to produce the SVG and PNG image files.